modules/tinymce/src/themes/silver/main/ts/ui/dialog/Dialogs.ts

```diff
@@ -121,8 +121,17 @@ export interface DialogSpec {
   extraStyles: Record<string, string>;
   dialogEvents: AlloyEvents.AlloyEventKeyAndHandler<any>[];
   eventOrder: Record<string, string[]>;
+  firstTabstop?: number;
 }
 
+const renderDialogWithHeader = (spec: DialogSpec): SketchSpec => {
+  return renderDialog({ ...spec, firstTabstop: 1 });
+};
+
+const renderDialogWithHiddenHeader = (spec: DialogSpec): SketchSpec => {
+  return renderDialog(spec);
+};
+
```

Comment:
Can't we just add `firstTabstop: 1` to `SilverDialogCommon` instead of changing `Dialog` directly?

Nothing about these methods guarantees there is a close button or not, as the name implies. So we might as well leave control in the hands of the calling function.

- [ ] consider simplifying

Reply:
Oh yeah, thanks for the suggestion. We could pass `firstTabstop: 1` from `SilverDialogCommon` and omit it if it was called from a function that doesn't require setting `firstTabstop`. Will make the changes and revert back to a single function in `Dialogs.ts`.